<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BCSettingMatrixDlg</class>
 <widget class="QDialog" name="BCSettingMatrixDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="title">
        <string>VGA 矩阵配置</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="m_pVGAMatrixCheckBox">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>VGA矩阵已连接</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>输入端口数：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>矩阵地址：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>通讯参数：</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLineEdit" name="m_pCommunicationParaVGALineEdit">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="m_pVGASetBtn">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>设置</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="m_pInputPortCountOfVGASpinBox">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="m_pMontageAddOfVGASpinBox">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QPushButton" name="m_pVGAPreSetBtn">
          <property name="maximumSize">
           <size>
            <width>137</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>设置前置级联矩阵</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="title">
        <string>视频矩阵配置</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QCheckBox" name="m_pVideoMatrixCheckBox">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>视频矩阵已连接</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>输入端口数：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>矩阵地址：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>通讯参数：</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLineEdit" name="m_pCommunicationParaVideoLineEdit">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="m_pVideoSetBtn">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>设置</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="m_pInputPortCountOfVideoSpinBox">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="m_pMontageAddOfVideoSpinBox">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QPushButton" name="m_pVideoPreSetBtn">
          <property name="maximumSize">
           <size>
            <width>137</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>设置前置级联矩阵</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="title">
        <string>DVI 矩阵配置</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="2">
         <widget class="QCheckBox" name="m_pDVIMatrixCheckBox">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>DVI矩阵已连接</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>输入端口数：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>矩阵地址：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>通讯参数：</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLineEdit" name="m_pCommunicationParaDVILineEdit">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="m_pDVISetBtn">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>设置</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="m_pMontageAddOfDVISpinBox">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="m_pInputPortCountOfDVISpinBox">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QPushButton" name="m_pDVIPreSetBtn">
          <property name="maximumSize">
           <size>
            <width>137</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>设置前置级联矩阵</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_4">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="title">
        <string>HDMI 矩阵配置</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QCheckBox" name="m_pHDMIMatrixCheckBox">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>HDMI矩阵已连接</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>输入端口数：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="m_pInputPortCountOfHDMISpinBox">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>矩阵地址：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="m_pMontageAddOfHDMISpinBox">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_12">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>通讯参数：</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLineEdit" name="m_pCommunicationParaHDMILineEdit">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="m_pHDMISetBtn">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>设置</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="m_pHDMIPreSetBtn">
          <property name="maximumSize">
           <size>
            <width>137</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>设置前置级联矩阵</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="groupBox_5">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="title">
        <string>SDI 矩阵配置</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" colspan="2">
         <widget class="QCheckBox" name="m_pSDIMatrixCheckBox">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>SDI矩阵已连接</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_13">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>输入端口数：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_14">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>矩阵地址：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_15">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>通讯参数：</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLineEdit" name="m_pCommunicationParaSDILineEdit">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="m_pSDISetBtn">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>设置</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="m_pInputPortCountOfSDISpinBox">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="m_pMontageAddOfSDISpinBox">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QPushButton" name="m_pSDIPreSetBtn">
          <property name="maximumSize">
           <size>
            <width>137</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>设置前置级联矩阵</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QGroupBox" name="groupBox_6">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="title">
        <string>音频矩阵配置</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0" colspan="2">
         <widget class="QCheckBox" name="m_pAudioMatrixCheckBox">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>音频矩阵已连接</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_16">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>输入端口数：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_17">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>矩阵地址：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_18">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>通讯参数：</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLineEdit" name="m_pCommunicationParaAudioLineEdit">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="m_pAudioSetBtn">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>设置</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QPushButton" name="m_pAudioPreSetBtn">
          <property name="maximumSize">
           <size>
            <width>137</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
          <property name="text">
           <string>设置前置级联矩阵</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="m_pInputPortCountOfAudioSpinBox">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="m_pMontageAddOfAudioSpinBox">
          <property name="font">
           <font>
            <family>微软雅黑</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pOkBtn">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pCancelBtn">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
